### DISCUSSION TITLE
maximum of repo count

### DISCUSSION THREAD

**Discussion by jankozik** [community/community#62858] [Repositories]
### Select Topic Area

Question

### Body

How many repositories can I create on my GitHub?

**Comment by goldest-star** [comment]
GitHub allows users to create an unlimited number of repositories. 
There is no specific limit on the number of repositories you can create on GitHub.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
